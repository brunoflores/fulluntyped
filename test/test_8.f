0;
succ (pred 0);
iszero (pred (succ 0));
iszero (pred (succ (succ 0)));
