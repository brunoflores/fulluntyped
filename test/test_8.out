0: Nat
1: Nat
true: Bool
false: Bool
